<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginWidget</class>
 <widget class="QWidget" name="CLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>754</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>754</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CLoginWidget{
background-color:rgb(18,18,18);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>741</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>741</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>741</width>
     <height>64</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#label{
 font-size:25px;
 color:rgb(224,224,224);
 
}</string>
      </property>
      <property name="text">
       <string>      允许控制本设备</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>701</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>701</width>
     <height>91</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>701</width>
     <height>91</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>701</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>701</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">#label_2{
color:rgb(106,106,106);
}</string>
         </property>
         <property name="text">
          <string>本设备识别码</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>218</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">#label_3{
color:rgb(106,106,106);
}</string>
         </property>
         <property name="text">
          <string>今日验证码</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>201</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>701</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>701</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>165</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>165</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>165</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_4{
color:rgb(224,224,224);
font-size:30px;
}</string>
       </property>
       <property name="text">
        <string>802 150 224</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>0</y>
         <width>21</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/IconLoginNum.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>0</y>
         <width>51</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_6{
color:rgb(71,71,71);
font-size:25px;
font-weight:bold;
}</string>
       </property>
       <property name="text">
        <string>  | </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>0</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit{
background-color:rgba(0,0,0,0);
color:rgb(224,224,224);
border:2px solid rgb(18,18,18);
}</string>
       </property>
       <property name="text">
        <string>123456</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>0</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/IconLoginSee.png</pixmap>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>723</width>
     <height>52</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>701</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>701</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>701</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_8{
color:rgb(224,224,224);
font-size:30px;
}</string>
      </property>
      <property name="text">
       <string>     远程控制设备</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>701</width>
     <height>76</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>701</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>701</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>44</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">#lineEdit_2{
border: 2px solid rgb(221,58,95);
width: 400px;
height: 50px;
background-color:rgb(18,18,18);
border-radius:10px;
color:rgb(106,106,106);
font-size:30px;
}</string>
      </property>
      <property name="placeholderText">
       <string>请输入识别码</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>14</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{
color:rgb(224,224,224);
widht:190px;
height:50px;
border:2px solid rgb(221,58,95);
background-color:rgb(221,58,95);
border-radius:10px;
}</string>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>701</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>701</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>701</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>345</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{
border:none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>95</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#radioButton{
color:rgb(224,224,224);
}

#radioButton:indicator:unchecked{
image:url(:/IconRadio.png);
}
#radioButton:indicator:checked{
image:url(:/IconRadio2.png);
}


#radioButton:hover{
color:rgb(221,52,84);
}

</string>
     </property>
     <property name="text">
      <string>远程桌面</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>95</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#radioButton_2{
color:rgb(224,224,224);
}

#radioButton_2:indicator:unchecked{
image:url(:/IconRadio.png);
}

#radioButton_2:indicator:checked{
image:url(:/IconRadio2.png);
}

#radioButton_2:hover{
color:rgb(221,52,84);
}</string>
     </property>
     <property name="text">
      <string>远程文件</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
